sdks/wasm/DebuggerTestSuite/Support.cs

```diff
@@ -167,6 +169,7 @@ public Task Ready ()
 					dbgUrl = arrStr[0] + "/" + arrStr[1] + "/" + arrStr[2] + "/" + arrStr[arrStr.Length - 1];
 					dicScriptsIdToUrl[script_id] = dbgUrl;
 					dicFileToUrl[dbgUrl] = args["url"]?.Value<string>();
+					event_order.Add(dicFileToUrl[dbgUrl]);
```

Comment:
Since, this tracks events, I would make this `event_order.Add ($"Debugger.scriptParsed#{dicFileToUrl[dbgUrl}");`